extension/src/popup/views/__tests__/ManageAssets.test.tsx

```diff
@@ -174,16 +174,19 @@ jest.mock("popup/constants/history", () => ({
 jest.mock("stellar-sdk", () => {
   const original = jest.requireActual("stellar-sdk");
   return {
-    ...original,
-    Server: class {
-      loadAccount() {
-        return {
-          sequenceNumber: () => 1,
-          accountId: () => publicKey,
-          incrementSequenceNumber: () => {},
-        };
-      }
+    Networks: original.Networks,
+    Horizon: {
+      Server: class {
+        loadAccount() {
+          return {
+            sequenceNumber: () => 1,
+            accountId: () => publicKey,
+            incrementSequenceNumber: () => {},
+          };
+        }
+      },
     },
+    SorobanRpc: original.SorobanRpc,
```

Comment:
fixing the structure on these SDK methods.

Also, there's a weird issue where `...original` wasn't working because there's something wrong with the Getter for `ContractSpec` 🤔 I'll investigate further, but it's probably best to disable any SDK methods we're not using to prevent unexpected calls to Horizon (more on this later)